### PR TITLE
[Fix] Made releasing-plugin.adoc sound better. 

### PR DESCRIPTION
**Pull request by ForBai**
## What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [x] Documentation
- [ ] Other, please describe:

## Does this PR introduce a breaking change?

- [x] No
- [ ] Yes

## Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/asciidoctor/asciidoctor-intellij-plugin/blob/main/CONTRIBUTING.adoc) document.
- [x] I have given my PR a descriptive name. If it resolves a specific issue, I referenced it in the form of `fix #xxx` (where `xxx` is the issue number).
- [x] I have updated the [documentation](https://github.com/asciidoctor/asciidoctor-intellij-plugin/blob/main/doc/users-guide/modules/ROOT/pages/features/) to reflect the changes introduced by this PR.
- [ ] I have added tests to cover the changes in the code.
- [ ] I need help writing a test (Maintainers are willing to help).

Creating the pull request triggers some automated tests.
See the [chapter building in the contributor's guide](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/coder/building-and-running.html) to find out how to run them on your local machine.

### CODE REVIEW

File: doc/contributors-guide/modules/ROOT/pages/coder/releasing-plugin.adoc [hunk 1]
```diff
@@ -25,7 +25,7 @@ See _xref::user/add-eap-repository-to-ide.adoc[]_ for more information.
 . Go to https://github.com/asciidoctor/asciidoctor-intellij-plugin/releases[GitHub releases for this plugin]
 . Edit the pre-release and uncheck the checkbox "`This is a pre-release`". +
 A GitHub action will then publish the release.
-You don't need to wait for the publishing process of the preview version to complete to be able to trigger publishing a stable version.
+It isn't needed to wait for the publishing process of the preview version to complete to be able to trigger publishing a stable version.
```
Comment (ahus1) [inline doc/contributors-guide/modules/ROOT/pages/coder/releasing-plugin.adoc:28]: Your change is valid, and an improvement to the original text. 

Reading the old and new text, I found that the negation (that was there before you change the text) is hard to read, and rewrote it a bit. The sentence is now also a bit shorter, what should make it easier to understand. What do you think?

```suggestion
Publishing of the stable version can start while the GitHub action publishing the the preview version is still running.
```

**Review by ahus1 — CHANGES_REQUESTED** [review]
Thank you for the docs update! I have a minor suggestion to one of the sentences. Please let me know what you think.

File: doc/contributors-guide/modules/ROOT/pages/coder/releasing-plugin.adoc (hunk 1 above)
Comment (ForBai) [reply doc/contributors-guide/modules/ROOT/pages/coder/releasing-plugin.adoc:28]: Your change sounds better to me, but there is a double the that would need to be removed tho.

**Comment by ahus1** [comment]
Thanks a lot for contributing this, and also spotting my mistake. This change will be online on the web site within minutes.

**Pull request merged**